pkg/function/create_csi_snapshot.go

```diff
@@ -143,7 +143,7 @@ func (*createCSISnapshotFunc) Arguments() []string {
 }
 
 func createCSISnapshot(ctx context.Context, snapshotter snapshot.Snapshotter, name, namespace, pvc, snapshotClass string, wait bool, labels map[string]string) (*v1.VolumeSnapshot, error) {
-	if err := snapshotter.Create(ctx, name, namespace, pvc, &snapshotClass, wait, labels); err != nil {
+	if err := snapshotter.Create(ctx, name, namespace, pvc, &snapshotClass, wait, labels, nil); err != nil {
```

Comment:
Instead of passing `nil` should we pass empty annotation `map[string]string{}`.

Reply:
Passed `nil` by getting the idea from how labels are implemented as below:
```
if labels != nil {
		snap.SetLabels(labels)
	}
	if annotation != nil {
		snap.SetAnnotations(annotation)
	}
```